// vega-lite-api/src/groupby.js
import {BaseObject, assign, flat, init, set} from './__util__.js';
import {aggregate} from './aggregate.js';
import {density} from './density.js';
import {join} from './join.js';
import {joinaggregate} from './joinaggregate.js';
import {loess} from './loess.js';
import {pivot} from './pivot.js';
import {quantile} from './quantile.js';
import {regression} from './regression.js';
import {window} from './window.js';

class Groupby extends BaseObject {

  constructor(...args) {
    super();
    init(this);
    args = flat(args);
    set(this, "groupby", args);
  }

  aggregate(...values) {
    const obj = aggregate(...values);
    return assign(obj, this);
  }

  density(...values) {
    const obj = density(...values);
    return assign(obj, this);
  }

  join(...values) {
    const obj = join(...values);
    return assign(obj, this);
  }

  joinaggregate(...values) {
    const obj = joinaggregate(...values);
    return assign(obj, this);
  }

  loess(...values) {
    const obj = loess(...values);
    return assign(obj, this);
  }

  pivot(...values) {
    const obj = pivot(...values);
    return assign(obj, this);
  }

  quantile(...values) {
    const obj = quantile(...values);
    return assign(obj, this);
  }

  regression(...values) {
    const obj = regression(...values);
    return assign(obj, this);
  }

  window(...values) {
    const obj = window(...values);
    return assign(obj, this);
  }

}

export function groupby(...args) {
  return new Groupby(...args);
}
